resources/chinese.yml

```diff
@@ -982,53 +982,1038 @@ msg_data_successfully_deleted: 'Data successfully deleted!'
 msg_err_key_cannot_be_deleted: 'Key cannot be deleted because it does not exist!'
 msg_err_this_town_doesnt_have_any_associated_metadata: 'This town doesn''t have any associated metadata.'
 msg_err_key_x_is_not_part_of_this_town: 'Key %s is not part of this town, if possible please add it.'
-msg_custom_data_was_successfully_added_to_town: 'Custom data was successfully added to town!'
-msg_error_too_many_pages: 'Too many pages.'
-msg_err_location_is_not_within_a_town: 'Location is not within a town.'
-msg_err_location_is_not_within_a_jail_plot: 'Location is not within a Jail plot.'
-msg_err_town_has_no_jail_spawns_set: 'Town has no jail spawns set.'
-msg_err_town_has_no_outpost_spawns_set: 'Town has no outpost spawns set.'
-msg_err_location_is_not_within_an_outpost_plot: 'Location is not within an outpost plot.'
-msg_err_town_has_not_set_a_spawn_location: 'Town has not set a spawn location.'
-msg_err_town_has_no_claim_over_this_town_block: 'Town has no claim over this town block.'
-msg_err_town_doesnt_belong_to_any_nation: 'Town doesn''t belong to any nation.'
-msg_err_mayor_doesnt_belong_to_town: 'Mayor doesn''t belong to town.'
-msg_err_tag_too_long: 'Tag too long.'
-msg_err_nation_has_not_set_a_spawn_location: 'Nation has not set a spawn location.'
-msg_err_resident_doesnt_belong_to_any_town: 'Resident doesn''t belong to any town'
-msg_err_could_not_set_default_modes_for: 'Could not set default modes for '
+name: Towny
```

Comment:
Your file has got the beginning in the middle. Please return the file to its proper order.